Detections/SophosXGFirewall/PortScanDetected.yaml

```diff
@@ -0,0 +1,27 @@
+id: 427e4c9e-8cf4-4094-a684-a2d060dbca38
+name: Port Scan Detected
+description: |
+  'This alert creates an incident when a source IP addresses attempt to communicate with a large amount of distinct ports within a short period.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: SophosXGFirewall
+    dataTypes: 
+      - Syslog
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Discovery
+relevantTechniques:
+  - T1046
+query: |
+
+  let timeframe = ago(1h);
+  SophosXGFirewall
```

Comment:
This would need to be Syslog as that is the dataType this is flowing into

Reply:
These Analytics Rules require a Kusto Function that enables the SophosXGFirewall table to be used. Should the dataType be changes from Syslog to SophosXFirewall?

Reply:
Yes, if the table is not Syslog, then the name of the table is what you would use here. Should be the same as the dataTypes value.

Reply:
Corrections have been made and pushed.